### ISSUE TITLE
Grab county-level shapes for country

### ISSUE THREAD

**Issue by Makosak** [GeoDaCenter/covid#1]

**Comment by linqinyu** [comment]
I tested an R package named "USAboundariesData". It worked. If we decide to go with R, I can add the code and shape file to github.

**Comment by lanselin** [comment]
make sure the county boundaries are compatible with the data. the boundary files change
over time and the boundaries in this package may not be up to date and/or not fully compatible
with the county list for the data. we have a few different county boundary files in the geoda sample
data.

L.

**Comment by linqinyu** [comment]
Hi Luc, 

Thanks for the suggestions!

I tried two packages ("tigris" and "USAboundariesData"). I compared both of their national county shapefiles (claimed to be "current version"). They have the same total number of counties and when plotting them together I did not see major differences. 

However, when I went to the official census bureau website and looked at the changes of counties over time (https://www.census.gov/programs-surveys/geography/technical-documentation/county-changes.2010.html), I found both packages only covered the changes up to 2010, but not **after** 2010.  Luckily, only 5 of the changes after 2010 are in the Lower 48. All the other changes are within Alaska. I guess we can hand-code these changes if no better alternative can be found. 

I also checked the Geoda sample datasets. But I did not find any dataset that covers all counties after 2000. Let me know if I miss anything. 

Thanks!

Best wishes,
Qinyun
